src/lib.rs

```diff
@@ -174,14 +174,31 @@ macro_rules! iproduct {
 /// # }
 /// ```
 macro_rules! izip {
-    ($I:expr) => (
-        (::std::iter::IntoIterator::into_iter($I))
-    );
-    ($($I:expr),*) => (
-        {
-            $crate::multizip(($(izip!($I)),*))
-        }
-    );
+    () => {
+        ::std::iter::empty::<()>()
+    };
```

Comment:
You're right – there's no `impl_zip_iter!()`. I somehow thought this case worked before. (also it should be `repeat` rather than `empty`)